css/main.css

```diff
@@ -1,3 +1,171 @@
+.hero {
+  background: url("../img/hero-background.png");
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+.hero div {
+  display: flex;
+  align-items: center;
+  font-size: 1.5rem;
+}
+.hero div p {
+  font-weight: 400;
+  padding-bottom: 0.5rem;
+  border-bottom: var(--white) 2px solid;
+}
+.hero div svg {
+  margin: 2rem 2rem 0;
+}
+
+.hero h3 {
+  font-size: 2rem;
+  font-weight: 400;
+  letter-spacing: 0.3rem;
+  text-transform: uppercase;
+  text-align: center;
+}
+.hero .button {
+  margin-bottom: 6rem;
+}
+.featured {
+  margin: 2rem -2rem 2rem;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 2rem;
+  align-items: center;
+  width: calc(100% + 4rem);
+}
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  object-fit: cover;
+}
+.product img {
+  width: 100%;
+  height: auto;
+}
+.product .product-name,
+.product .underline,
+.product a {
+  position: absolute;
+  top: 2rem;
+}
+.product a {
+  position: absolute;
+  top: 7rem;
+}
+.product-name {
+  top: 2rem;
+  text-decoration: none;
+  font-size: 2rem;
+  letter-spacing: 0.4rem;
+  color: var(--red-brown);
+  transition: 0.3s all ease-in;
+}
+.product-name:hover {
+  color: var(--black);
+}
+.product .underline {
+  top: 3.4rem;
+  font-size: 3rem;
+}
+.about-us {
+  scroll-snap-align: start;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 0;
+}
+.about-us > * {
+  grid-column: auto / span 4;
+}
+.about-us img {
+  width: calc(100% + 4rem);
+  margin: -0.2rem -2rem;
+}
+.about-text,
+.special-offer {
+  height: 100%;
+  width: calc(100% + 4rem);
+  margin: 0 -2rem;
+  padding: 3rem 2rem;
+  text-align: center;
+}
+.about-text {
+  background-color: var(--white);
+  color: var(--red-brown);
+}
+.special-offer {
+  padding: 4rem;
+  font-size: 3rem;
+  font-weight: 800;
+  color: var(--white);
+  background-color: var(--green-black);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+.special-offer .button {
+  margin-top: 3rem;
+  background-color: var(--white);
+  color: var(--green-black);
+  transition: all 0.4s ease-in-out;
+}
+.special-offer .button:hover,
+.subscription-box .button:hover {
+  background-color: var(--green-light);
+  color: var(--white);
+}
+.about-text h2 {
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  letter-spacing: 0.5rem;
+  font-family: var(--heading-font);
+  font-weight: 400;
+}
+.about-text p {
+  font-size: 1.2rem;
+  line-height: 1.7rem;
+  margin-top: 1.5rem;
+}
+@media screen and (min-width: 900px) {
+  .featured {
+    padding: 0 2rem;
+    grid-template-columns: repeat(3, 1fr);
+    height: 60vh;
+    width: calc(100% + 4rem);
+    grid-gap: 2rem;
+  }
+  .product {
+    height: 60vh;
+    overflow: hidden;
+  }
+  .product img {
+    height: 100%;
+    width: auto;
+  }
+  .about-us {
+    grid-template-rows: 1fr 1.3fr;
+    height: 150vh;
```

Comment:
Best not to set fixed heights unless you really need to. In this case, I don't think you do, and it's breaking the contact form layout a little at the start of your desktop view (between 900px and 1160px approx)

Reply:
Other than that, great use of Grid on this section 👌